GoogleForCreators/web-stories-wp
# KSES: Extend allowlist to fix templates usage

## Context



Templates like the Summer Fashion one were broken when used by a user without `unfiltered_html` capabilities.

## Summary



Extend KSES allowlist to fix templates usage for these users.

## Relevant Technical Choices



## To-do



## User-facing changes



<table>
<tr>
	<td>Before
	<td>After
<tr>
	<td><img src="https://user-images.githubusercontent.com/841956/135598792-a56af6ad-382f-48dc-a76f-c56170d62185.png" alt="Actual result" />
	<td><img src="https://user-images.githubusercontent.com/841956/135598777-e9dd6eee-fc35-448e-9782-cd4a57a28356.png" alt="Expected appearance" />
</table>

## Testing Instructions






This PR can be tested by following these steps:

1. Log in as author
2. Create new story from Summer Fashion template
3. Preview story
4. Verify that the preview looks (roughly?) like in the screenshots and there are no broken elements or animations
5. Repeat with 2-3 other templates from the newer ones


## Reviews

### Does this PR have a security-related impact?



Hardening KSES

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #9227
